pkg/controller/controller_binding_test.go

```diff
@@ -77,6 +78,10 @@ func TestReconcileServiceInstanceCredentialNonExistingServiceInstance(t *testing
 	updatedServiceInstanceCredential := assertUpdateStatus(t, actions[0], binding)
 	assertServiceInstanceCredentialReadyFalse(t, updatedServiceInstanceCredential, errorNonexistentServiceInstanceReason)
 
+	assertServiceInstanceCredentialCurrentOperationClear(t, updatedServiceInstanceCredential)
```

Comment:
In a follow-up, it seems like we could collapse each of these assertions into a single `assertServiceInstanceCredentialFullyReconciled`; same thing for the equivalent checks for `ServiceInstance`.

Reply:
Agree. It's been on my list of things to do.